### PR TITLE
Feat/cleanup

### LINKED ISSUE #85: Remove DatabaseTestController and dev routes
Remove
- [x] DatabaseTestController
- [x] Dev routes

### PR DESCRIPTION
**Pull request by BenediktHaas96**
Closes #85 

Removes many empty files (seeders and controllers primarily) and cleans up some old, unused or unwanted code

### CODE REVIEW

**Review by TitusKirch — APPROVED** [review]

File: .github/workflows/release-chart.yml
Comment (simonostendorf) [inline .github/workflows/release-chart.yml:1]: This CI is not in our dev branch and should not be readded here.

**Review by simonostendorf — CHANGES_REQUESTED** [review]
can not verify that all removed controllers and seeders are not needed anymore but found a readded github action that should be removed here.

File: .github/workflows/release-chart.yml
Comment (BenediktHaas96) [reply .github/workflows/release-chart.yml:1]: Merged in dev branch, issue should be resolved now. Please verify

Comment (simonostendorf) [reply .github/workflows/release-chart.yml:1]: Solved.

**Review by simonostendorf — APPROVED** [review]

**Pull request merged**